src/dataEditor/dataEditorClient.ts

```diff
@@ -385,7 +385,7 @@ export class DataEditorClient implements vscode.Disposable {
         }
         break
 
-      case MessageCommand.commit:
+      case MessageCommand.apply:
```

Comment:
Given that we're going to change the term, I think "Apply" would suffice for the button label but maybe we should make the source code a little more verbose?

`MessageCommand.applyEdits` or `applyChangesToViewport`

I just wonder if we'll end up having to change the term in the source if we choose a generic `apply`.

Reply:
It's not just applying edits to the viewport, but to the edit model in Ωedit.

Reply:
Your requested change has been made to the message enum.